features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/apipublisher/scripts/apipublisher.js

```diff
@@ -110,8 +110,22 @@ var apipublisher = {};
         return this.impl.getAPIsByProvider(providerName);
     };
 
-    APIProviderProxy.prototype.getSubscribersOfAPI = function (apiId) {
-        return this.impl.getSubscribersOfAPI(apiId);
+    /*
+     * This method returns the subscription details needed for api-subscriptions page.
+     */
+    APIProviderProxy.prototype.getSubscribersOfAPI = function (provider, name, version) {
+        var apiObj = new Packages.org.json.simple.JSONObject();
+        apiObj.put("provider", provider);
+        apiObj.put("name", name);
+        apiObj.put("version", version);
```

Comment:
Thanks for pointing out this. This is fixed in https://github.com/thushara35/carbon-apimgt/commit/9eae1ae859d39638737d722a0df0b51b1eaca305